src/structures/cl_args.cpp

```diff
@@ -26,13 +27,27 @@ void update_host(Servers& servers, const std::string& value) {
     host = value.substr(index + 1);
   }
 
+  // remove any trailing slash
+  if (host.find('/') == host.length() - 1) {
+    host.pop_back();
+  }
```

Comment:
This would fail to remove the trailing slash with multiple occurrences, say for a host like `routing.openstreetmap.de/the/osrm/slug/routed-car/`. We should probably rather simply check the value of `host.back()` (if `host` is not empty).

Reply:
of course, my bad